docker/rust_verification/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM rust_klee:v3.0
```

Comment:
Is this necessary to be able to use the tools or is it just a demo?

I am new to Docker and do not know the usual ways of using Docker but I was hoping that we could package up the tools but still use them as if they were installed natively.
So I was expecting to see a Dockerfile that packaged up scripts/cargo-verify but I was not expecting one for try_klee

Reply:
This file is just a demo and is not necessary to be able to use the tools. What these dockerfiles and scripts try to build is more likely to be a development environment for rust-verification-tools.
Do you mean what you want is that when you have a native crate, for example ,hello , than you can run , `docker run rust-verification-tools  hello` to use scripts/cargo-verify in docker to verify it? It needs a small fix, and I can push a commit later for it.